apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -852,6 +863,40 @@ sophia_identity(_Cfg) ->
     77    = ?call(call_contract,   Acc1, RemC2, staged_call, word, {IdC, RemC, 77}),
     ok.
 
+sophia_exploits(_Cfg) ->
```

Comment:
I tested locally that this commit with test without the related code changes makes the test fail (took veeeeery long time). Ok.